### DISCUSSION TITLE
Is it possible manually active breakpoint style?

### DISCUSSION THREAD

**Discussion by wenerme** [tailwindlabs/tailwindcss#3282] [Help] [answered]
Tring to implement a window style react app, the window container is resizeable, want to use the tailwindcss for the responsive style, but active the style by the window div size, something like this

```jsx
<WindowContainer>
  <div className="p-2 md:p-4 lg:p-8">
    content
  </div>
</WindowContainer>
```

WindowContainer is resizeable, can change the class based on size.

Is it possible to active the `md` or `lg` based on the parent class ? or how to achieve this ?

**Comment by simonswiss** [comment, marked as answer]
Hey!

Unfortunately, until "container queries" becomes a thing, it's pretty complicated to achieve what you're describing.

The [ResizeObserver API](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver) is something you can use, paired with the use of an `iframe`, to achieve this.
